tracing-subscriber/src/fmt/mod.rs

```diff
@@ -124,6 +124,8 @@
 use std::{any::TypeId, error::Error, io};
 use tracing_core::{span, subscriber::Interest, Event, Metadata};
 
+#[cfg(not(feature = "chrono"))]
+mod datetime;
```

Comment:
nit/take it or leave it: since this is used _only_ in the `time` module, it could probably be a submodule of `time` rather than of `fmt`. but, not a big deal.